### DISCUSSION TITLE
shift to mlops

### DISCUSSION THREAD

**Discussion by nakytoe** [City-of-Helsinki/mlops-template#50] [Announcements]
## What's Changed
- Change of name from ml_project_template to mlops-template, because of the clarified use case of API-streamed ML. Exploratory work now has a [separate repository template](https://github.com/Datahel/analytics_template) with identical container base for compatibility.
- Refined structure, lighter documentation, single-container
- Upgrade to nbdev2, but focus nbdev use for ML model development. Less-prone-to-change code parts are now defined in .py files
- New features: api, prometheus metrics, logging, model store
- Improved model & experiment versioning
- Environment options: Codespaces, VSC, jupyterlab & offline install
- Ethical principles and issues
- Working examples

**Full Changelog**: https://github.com/City-of-Helsinki/mlops-template/compare/v0.0.1...v0.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/City-of-Helsinki/mlops-template/releases/tag/v0.0.2'>shift to moles</a>.</em>

**Comment by petterill** [comment]
That link leads to a private repo.

**Reply by nakytoe** [reply]
Yes, the analytics template repository is currently private. It will be published later
